src/SceneView.tsx

```diff
@@ -110,7 +110,7 @@ export default class SceneView<T extends Route> extends React.Component<
             : focused
             ? StyleSheet.absoluteFill
             : null,
-          style,
+          containerStyle,
```

Comment:
I ran into an edge case with this if you want to add a tab view that's not a full screen dimension.
if you add back the `styles` prop and add this line `!focused && style` right above `containerStyle`, it will allow the active scene content to dictate the height (unless you've set height on `TabView` styles)

```suggestion
          !focused && defaultStyle,
          containerStyle,
```


https://user-images.githubusercontent.com/3317835/124084979-67bb8900-da04-11eb-842b-98980a40a798.mp4

Reply:
Just so you know, without the suggested change. the heights will be messed up if you have multiple scenes

Reply:
This seems to cause performance degradation/ getting stuck halfway between tabs. Does anyone have thoughts?